tests/unit/matrix/test_subtotals.py

```diff
@@ -200,6 +201,102 @@ def it_can_compute_a_subtotal_row_to_help(self, request):
         )
 
 
+class DescribePairwiseSigTestSubtotals(object):
+    """Unit test suite for `cr.cube.matrix.PairwiseSigTestSubtotals` object."""
+
+    def it_provides_a_blocks_interface_method(self, request, _init_):
+        property_mock(
+            request,
+            PairwiseSigTestSubtotals,
+            "_blocks",
+            return_value=np.array([[[1], [2]], [[3], [4]]]),
+        )
+
+        blocks = PairwiseSigTestSubtotals.blocks(None, None, None, None, 0)
+
+        _init_.assert_called_once_with(ANY, None, None, None, None, 0, True, True)
+        assert blocks.tolist() == [[[1], [2]], [[3], [4]]]
+
+    def it_can_compute_a_intersection_cell_value_to_help(self):
+        assert np.isnan(
+            PairwiseSigTestSubtotals(None, None, None, None, 0)._intersection(
+                None, None
+            )
+        )
+
+    @pytest.mark.parametrize(
+        ("addend_idxs", "subtrahend_idxs", "diff_cols_nan", "expected_value"),
+        (
+            ([1, 2], [], False, [[1], [2]]),
+            ([], [1, 2], False, [[1], [2]]),
+            ([0], [2], True, [[np.nan], [np.nan], [np.nan]]),
+        ),
+    )
+    def it_can_compute_a_subtotal_column_to_help(
+        self,
+        request,
+        subtotal_,
+        addend_idxs,
+        subtrahend_idxs,
+        diff_cols_nan,
+        expected_value,
+    ):
+        subtotal_.addend_idxs = addend_idxs
+        subtotal_.subtrahend_idxs = subtrahend_idxs
+        property_mock(request, PairwiseSigTestSubtotals, "_nrows", return_value=3)
+        subtotals = PairwiseSigTestSubtotals(
+            None, None, None, None, 0, diff_cols_nan, None
+        )
+        column = np.array([1, 2])
+
+        assert subtotals._subtotal_column(subtotal_, column) == pytest.approx(
```

Comment:
Maybe put the actual call on a separate line, before the assertion, so it's more easily detectable.